docs/package.json

```diff
@@ -65,7 +65,8 @@
     "canvas": "^2.9.1",
     "dotenv-safe": "^8.2.0",
     "esbuild-register": "^3.3.2",
-    "eslint-config-next": "12.0.4",
+    "eslint": "^7.32.0",
```

Comment:
eslint@^8 is incompatible with `next.js@11`, so we're using `^7.32.0` here.

https://github.com/vercel/next.js/issues/29829